optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc

```diff
@@ -594,7 +594,7 @@ On `UniConstraintStream`, the `ifExistsOther()` building block is also available
 `from()` constraint match type is the same as the `ifExists()` type.
 
 Conversely, if the `ifNotExists()` building block is used (as well as the `ifNotExistsOther()` building block on
-`UniConstraintStream`) you can achieve the opposite affect:
+`UniConstraintStream`) you can achieve the opposite effect:
```

Comment:
Not 100% sure but I believe this is a mistake. Affect seems to be exclusively tied to emotions: https://www.merriam-webster.com/dictionary/affect. @triceo please confirm.

Reply:
Yes, correct.